style/style.css

```diff
@@ -82,29 +87,201 @@ h2 {
   background-image: url(../assets/background.png);
   background-repeat: no-repeat;
   background-size: 375px 668px;
-  padding-left: 10px;
+  background-color: rgb(255, 255, 255);
   width: 375px;
   height: 718px;
+  margin: 0 auto;
+}
+
+.content ul,
+li {
+  list-style: none;
+  display: inline;
+  padding: 1px 13px;
+  margin: -3px;
 }
 
-ul,
+.contact ul,
 li {
   list-style: none;
   display: inline;
   padding: 1px 13px;
-  margin-left: -3px;
+  margin: -3px;
+}
```

Comment:
- [ ] This list here and around needs a few adjustments as well on padding and margin.
Also, take a quick look at the two images down below!
### **Your design**
![getmyresume](https://user-images.githubusercontent.com/47625407/120863803-8193b880-c58b-11eb-9071-bdcfcf6fe495.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)**
![originalgetmyresume](https://user-images.githubusercontent.com/47625407/120864325-578ec600-c58c-11eb-8aef-9b520f1ababb.png)

Reply:
:heavy_check_mark: Fixed.